rebar.config

```diff
@@ -11,7 +11,7 @@
         {relx,                "3.27.0"},
         {cf,                  "0.2.2"},
         {cth_readable,        "1.4.2"},
-        {hex_core,            "0.2.0"},
+        {hex_core,            {git, "https://github.com/hexpm/hex_core.git"}},
```

Comment:
This will use the version from hex_core after this PR is accepted and a new hex_core version is released.